tests/conftest.py

```diff
@@ -481,25 +491,24 @@ def execute_request(undo, context, client):
         # Instead of finding the response class of the method, we use the fact that all
         # responses returned have an ordered response of body|status|headers
         api_request["response"] = [e.body, e.status, e.headers]
-
-    api = api_request["api"]
-    operation_id = api_request["request"].settings["operation_id"]
-    response = api_request["response"][0]
-
-    context["undo_operations"].append(lambda: undo(api, operation_id, response))
+    else:
```

Comment:
Can you use `return` in exception block? The `else:` after `except:` is tricky for many non-python devs.

Reply:
Done